src/psyclone/tests/domain/lfric/lfric_field_test.py

```diff
@@ -91,6 +96,22 @@
 '''
 
 
+def test_ad_fld_type_1st_arg():
```

Comment:
I think this is a step in the right direction but we're still ending up with a test file that is very big. I think we need to separate out those tests that are purely meta-data related and those that test the code generation.

Reply:
Split the field tests into `lfric_field_mdata_test.py` and `lfric_field_codegen_test.py`.